tt_eager/tt_dnn/op_library/backward/backward_ops.cpp

```diff
@@ -1532,12 +1532,29 @@ std::vector<Tensor> binary_gt_bw(const Tensor& grad, const Tensor& input, const
     return operation::decorate_as_composite(__func__, _binary_gt_bw)(grad, input, output_mem_config);
 }
 
+// Autoformat support
+Tensor change_layout_to_tile(const Tensor& temp, const MemoryConfig& output_mem_config) {
+    auto formatted_input_tensor = temp;
+    if(formatted_input_tensor.get_layout()==Layout::ROW_MAJOR){
+        auto a_pad_shape = AutoFormat::pad_to_tile_shape(temp.get_legacy_shape(), false, false, true, true);
+        auto out_shape = temp.get_legacy_shape();
+        out_shape = {out_shape[0], out_shape[1], out_shape[2], out_shape[3]};
```

Comment:
What is the purpose of `out_shape = {out_shape[0], out_shape[1], out_shape[2], out_shape[3]};`?

Reply:
I've removed out_shape.